app/models/foobar.rb

```diff
@@ -0,0 +1,7 @@
+class Foobar < ActiveRecord::Base
```

Comment:
[IrresponsibleModule](https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md): has no descriptive comment
Lines: [1]

[UncommunicativeVariableName](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md): has the variable name 'a'
Lines: [3]

[UncommunicativeVariableName](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md): has the variable name 'b'
Lines: [4]

[UtilityFunction](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md): doesn't depend on instance state (maybe move it to another class?)
Lines: [2]